### ISSUE TITLE
kubernetes init installs *latest* instead of the actual latest version

### ISSUE THREAD

**Issue by shivamkm07** [dapr/cli#1104]
## Expected Behavior

`dapr init -k` installs *latest* dapr runtime instead of the actual latest runtime version (currently 1.9.0).

## Actual Behavior

`dapr init -k` installs actual latest dapr runtime (currently 1.9.0). 

## Release Note

RELEASE NOTE:

**Comment by mukundansundar** [comment]
This is the case when `dapr init` without runtime version is given ... when `dapr init --runtime-version <ver>` is given, it works as expected ...

**Comment by pravinpushkar** [comment]
```
➜  cli git:(fix_1) ✗ ldapr init -k
⌛  Making the jump to hyperspace...
ℹ️  Note: To install Dapr using Helm, see here: https://docs.dapr.io/getting-started/install-dapr-kubernetes/#install-with-helm-advanced

ℹ️  Container images will be pulled from Docker Hub
✅  Deploying the Dapr control plane to your cluster...
✅  Success! Dapr has been installed to namespace dapr-system. To verify, run `dapr status -k' in your terminal. To get started, go here: https://aka.ms/dapr-getting-started
➜  cli git:(fix_1) ✗ ldapr status -k
  NAME                   NAMESPACE    HEALTHY  STATUS   REPLICAS  VERSION  AGE  CREATED
  dapr-sidecar-injector  dapr-system  True     Running  1         1.9.0    1m   2022-10-14 12:37.30
  dapr-placement-server  dapr-system  True     Running  1         1.9.0    1m   2022-10-14 12:37.30
  dapr-dashboard         dapr-system  True     Running  1         0.11.0   1m   2022-10-14 12:37.30
  dapr-sentry            dapr-system  True     Running  1         1.9.0    1m   2022-10-14 12:37.30
  dapr-operator          dapr-system  True     Running  1         1.9.0    1m   2022-10-14 12:37.30
➜  cli git:(fix_1) ✗ ldapr upgrade -k --runtime-version 1.8.0
ℹ️  Container images will be pulled from Docker Hub
ℹ️  Dapr control plane version 1.9.0 detected in namespace dapr-system
ℹ️  Starting upgrade...
ℹ️  Downgrade detected, skipping CRDs.
✅  Dapr control plane successfully upgraded to version 1.8.0. Make sure your deployments are restarted to pick up the latest sidecar version.
➜  cli git:(fix_1) ✗ ldapr status -k
  NAME                   NAMESPACE    HEALTHY  STATUS   REPLICAS  VERSION  AGE  CREATED
  dapr-sentry            dapr-system  True     Running  1         1.8.0    16s  2022-10-14 12:39.20
  dapr-dashboard         dapr-system  True     Running  1         0.10.0   16s  2022-10-14 12:39.20
  dapr-sidecar-injector  dapr-system  True     Running  1         1.8.0    16s  2022-10-14 12:39.20
  dapr-operator          dapr-system  True     Running  1         1.8.0    16s  2022-10-14 12:39.20
  dapr-placement-server  dapr-system  True     Running  1         1.8.0    15s  2022-10-14 12:39.21
```

**Comment by shivamkm07** [comment]
./dapr init -k
```
⌛  Making the jump to hyperspace...
ℹ️  Note: To install Dapr using Helm, see here: https://docs.dapr.io/getting-started/install-dapr-kubernetes/#install-with-helm-advanced

ℹ️  Container images will be pulled from Docker Hub
✅  Deploying the Dapr control plane to your cluster...
✅  Success! Dapr has been installed to namespace dapr-system. To verify, run `dapr status -k' in your terminal. To get started, go here: https://aka.ms/dapr-getting-started
```
 ./dapr status -k
```
  NAME                   NAMESPACE    HEALTHY  STATUS   REPLICAS  VERSION  AGE  CREATED              
  dapr-operator          dapr-system  False    Running  1         1.9.0    3s   2022-10-14 12:38.07  
  dapr-placement-server  dapr-system  False    Running  1         1.9.0    2s   2022-10-14 12:38.08  
  dapr-sentry            dapr-system  False    Running  1         1.9.0    3s   2022-10-14 12:38.07  
  dapr-sidecar-injector  dapr-system  False    Running  1         1.9.0    3s   2022-10-14 12:38.07  
  dapr-dashboard         dapr-system  True     Running  1         0.11.0   3s   2022-10-14 12:38.07  
``` 
./dapr init -k --runtime-version 1.8.5

```
⌛  Making the jump to hyperspace...
ℹ️  Note: To install Dapr using Helm, see here: https://docs.dapr.io/getting-started/install-dapr-kubernetes/#install-with-helm-advanced

ℹ️  Container images will be pulled from Docker Hub
✅  Deploying the Dapr control plane to your cluster...
✅  Success! Dapr has been installed to namespace dapr-system. To verify, run `dapr status -k' in your terminal. To get started, go here: https://aka.ms/dapr-getting-started
```

./dapr status -k 
```
  NAME                   NAMESPACE    HEALTHY  STATUS   REPLICAS  VERSION  AGE  CREATED              
  dapr-sidecar-injector  dapr-system  True     Running  1         1.8.5    51s  2022-10-14 12:43.24  
  dapr-dashboard         dapr-system  True     Running  1         0.10.0   51s  2022-10-14 12:43.24  
  dapr-operator          dapr-system  True     Running  1         1.8.5    51s  2022-10-14 12:43.24  
  dapr-placement-server  dapr-system  True     Running  1         1.8.5    51s  2022-10-14 12:43.24  
  dapr-sentry            dapr-system  True     Running  1         1.8.5    51s  2022-10-14 12:43.24  
```

**Comment by shivamkm07** [comment]
Closed by #1103
